services/horizon/internal/ingest/processors/asset_stats_set.go

```diff
@@ -106,7 +118,20 @@ func (value assetStatValue) ConvertToHistoryObject() history.ExpAssetStat {
 }
 
 // AssetStatSet represents a collection of asset stats
```

Comment:
suggestion to enrich the docs with the structure change,

```suggestion
// AssetStatSet represents a collection of asset stats and a mapping 
// of Soroban contract IDs to classic assets (which is unique to each 
// network).
```